test/travis_ci.sh

```diff
@@ -185,39 +188,37 @@ function generate_ydktest_package {
 # sanity tests
 function run_sanity_ncclient_tests {
     printf "\nRunning sanity tests on NCClient client\n"
-    run_test sdk/python/tests/test_sanity_types.py
-    run_test sdk/python/tests/test_sanity_errors.py
-    run_test sdk/python/tests/test_sanity_filters.py
-    run_test sdk/python/tests/test_sanity_levels.py
-    run_test sdk/python/tests/test_sanity_filter_read.py
-    run_test sdk/python/tests/test_sanity_netconf.py
-    run_test sdk/python/tests/test_sanity_rpc.py
-    run_test sdk/python/tests/test_sanity_delete.py
-    run_test sdk/python/tests/test_sanity_service_errors.py
+    run_test gen-api/python/tests/test_sanity_types.py
+    run_test gen-api/python/tests/test_sanity_errors.py
+    run_test gen-api/python/tests/test_sanity_filters.py
+    run_test gen-api/python/tests/test_sanity_levels.py
+    run_test gen-api/python/tests/test_sanity_filter_read.py
+    run_test gen-api/python/tests/test_sanity_netconf.py
+    run_test gen-api/python/tests/test_sanity_rpc.py
+    run_test gen-api/python/tests/test_sanity_delete.py
+    run_test gen-api/python/tests/test_sanity_service_errors.py
 }
 
 function run_sanity_native_tests {
     printf "\nRunning sanity tests on native client\n"
-    ./test/test_native_ydk.py
-    run_test sdk/python/tests/test_sanity_types.py native
-    run_test sdk/python/tests/test_sanity_errors.py native
-    run_test sdk/python/tests/test_sanity_filters.py native
-    run_test sdk/python/tests/test_sanity_levels.py native
-    run_test sdk/python/tests/test_sanity_filter_read.py native
-    run_test sdk/python/tests/test_sanity_netconf.py native
-    run_test sdk/python/tests/test_sanity_rpc.py native
-    run_test sdk/python/tests/test_sanity_delete.py native
-    run_test sdk/python/tests/test_sanity_service_errors.py native
-    run_test sdk/python/tests/test_ydk_client.py
+    run_test gen-api/python/tests/test_sanity_types.py native
```

Comment:
Are we still gathering coverage for these files by running these tests from gen-api? The reason why these had to be run from sdk/python is that the files in sdk are checked into github and so the coveralls tool can gather the coverage and show it on coveralls.io